src/main/resources/edu/cornell/kfs/module/ar/cu-ar-resources.properties

```diff
@@ -1,3 +1,4 @@
 cginvoice.email.subject=Submission for Invoice Number: {0} / Agreement Number: {1} / Cornell OSP Number: {2}
 cginvoice.email.subject.nograntnumber=Submission for Invoice Number: {0} / Cornell OSP Number: {1}
-cginvoice.email.body=Dear {0}, \n\nAttached is Cornell University''s invoice for the subject award. \n\n**PLEASE DO NOT REPLY TO THIS ADDRESS** \n\nShould there be any questions, please contact the individual listed at the bottom of this invoice or at the contact information below. \n\nThank you, \n{1} \n{2} \n{3} 
\ No newline at end of file
+cginvoice.email.body=Dear {0}, \n\nAttached is Cornell University''s invoice for the subject award. \n\n**PLEASE DO NOT REPLY TO THIS ADDRESS** \n\nShould there be any questions, please contact the individual listed at the bottom of this invoice or at the contact information below. \n\nThank you, \n{1} \n{2} \n{3}
+invoice.address.pdf.final.watermark=FINAL 
```

Comment:
There is a space at the end of the line that is not present in base code.
Should that space be removed?

Reply:
good catch.  I don't the space will hurt anything, but it would be better to remove it.